src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,93 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.DuplicateAnExistingLoginException;
+import core.basesyntax.exceptions.NotCorrectAgeException;
+import core.basesyntax.exceptions.NotCorrectPassword;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    private static final User anna = new User();
+    private final RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
+
+    @BeforeAll
+    static void beforeAll() {
+        anna.setAge(18);
+        anna.setId(15893234L);
+        anna.setLogin("Anna@gmaol.com");
+        anna.setPassword("123456789");
+    }
+
+    @BeforeEach
+    void beforeEach() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void correctUserCheck_Ok() {
```

Comment:
There are a lot of ways to name your test methods. The main point is that they should have informative names and be consistent along with other developers in your team. For this task use such convention: `<methodUnderTest>_<state>_<expectedBehavior>`; For example, if we are testing the method register with a `null` user's age the test method name should be `register_nullAge_notOk`. `notOk` is because the test expects the register method to throw an exception.
```suggestion
    void register_validUser_Ok() {
```

same for other tests